docs/setup/using_in_browser.md

```diff
@@ -0,0 +1,10 @@
+[Initialization of library](./intialization.md) requires several properties of [signer](../api/classes/modules_signer.SignerService.md):
+
+- The signer meets [EIP-191: Signed Data Standard](https://eips.ethereum.org/EIPS/eip-191)
+- Public key. This is legacy parameter, which was used to initialized DID document
+
+These properties are not directly exposed by all signers, such as Metamask, and thus are determined in runtime by sending test sign request.
+In order to improve user expirience when library is used in browser it tries to get these parameters from local storage.
+Because library doesn't change storage application should set `isEthSigner` and `PublicKey` storage variables on login.
+Library doesn't send sign request if properties are not in local storage.
```

Comment:
Shouldn't it be the other way around. The library **does** send the request if properties are not in local storage?

Reply:
Yes, it is send request, but not as part of signer initialization. `isEthrSigner` and `publicKey` are calculated as part of login to cache server. I'll make this more clear

Reply:
Maybe we want to change that and initialize signer completely before login to ssi-hub